bambi/plots/plot_cap.py

```diff
@@ -133,9 +150,12 @@ def plot_cap(model, idata, covariates, use_hdi=True, hdi_prob=None, legend=True,
         Changing the global variable ``az.rcParam["stats.hdi_prob"]`` affects this default.
     legend : bool, optional
         Whether to automatically include a legend in the plot. Defaults to ``True``.
+    transforms : dict, optional
+        Transformations that are applied to each of the variables being plotted. The keys are the
+        name of the variables, and the values are functions to be applied. Defaults to ``None``.
```

Comment:
```suggestion
        name of the variables, and the values are functions to be applied. Defaults to `None`.
```

Reply:
Actually, just curious why are the double ticks needed? I see it in other docstrings too.

Reply:
If I remember correctly it's because it doesn't render properly if it only has a single backtick. Maybe something changed and now it works with a single backtick, but previously I remember trying a single backtick without working